.github/workflows/cyclonus-netpol-test.yaml

```diff
@@ -17,6 +17,10 @@ on:
 jobs:
   cyclonus-test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # run cyclonus tests in parallel for NPM with the given ConfigMaps
+        profile: [v1-default.yaml, v2-default.yaml]
```

Comment:
Can you remove v2-default for now and have the placefirst profile here, then we can merge this, when required and when we have all green we can enable v2-profile too. Wdyt ? 

I am okay to merge with v2-default and get some results but the master branch will show failures.